### DISCUSSION TITLE
UNO android app not receiving mqttclient subscribed topic

### DISCUSSION THREAD

**Discussion by LiftTrack** [unoplatform/uno#11922] [Q&A]
### Current behavior

Hi,

I am using MQTTnet in my Android app. I have created an Android application on the UNO platform. The application is working fine in debug mode but when I publish my app and run it on my mobile phone, I am unable to **receive subscribed topic** after installing "APK" file on my mobile phone. Unable to find out the cause behind it.

Steps:-
1. I am connected with "**test.mosquitto.org**".I am able to receive messages when I published anything under a particular topic.
2. In release mode or by installing the published "**APK**" file on my mobile phone, the app is not receiving published messages from "**test.mosquitto.org**".

Please provide me with a solution to it. Why my app is not receiving published messages in release mode?

Attaching a video for help.

https://user-images.githubusercontent.com/119324311/230412740-56e1b825-91ea-4cb8-a1e0-1d496416bd9e.mp4

https://user-images.githubusercontent.com/119324311/230412785-dd5c168f-b5ff-4b32-8d4f-5c7eb5dc200c.mp4

### Expected behavior

My App should work in the same way as it is working in debug mode. But after installing published "APK", App is not able to receive MQTTnet messages.

### Works on UWP/WinUI

None

### Environment

Other

### NuGet package version(s)

https://github.com/dotnet/MQTTnet--4.1.4.563

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

17.5.0

### Relevant plugins

https://github.com/dotnet/MQTTnet

### Anything else we need to know?

You should use "https://github.com/chkr1011/MQTTnetApp" application to get connected with "**test.mosquitto.org**"

**Comment by jeromelaban** [comment]
You may be hitting a configuration issue with the linker settings. Make sure to review https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options?pivots=dotnet-7-0.

**Reply by LiftTrack** [reply]
I have used your suggestions but it is not working .Attaching screenshot.

![ImplementedIssue](https://user-images.githubusercontent.com/119324311/230852976-0570d0e5-ed97-430b-b8df-882546afa416.png)

Thanks
Sanjay

**Reply by jeromelaban** [reply]
You may want to try and disable the linker altogether and see if it helps. If it does, you'll need to find the right linker configuration to remove some assemblies from the linker.
